### DISCUSSION TITLE
Request to provide different UML diagram like Class , architectural diagram to understand the project

### DISCUSSION THREAD

**Discussion by ajayjadhav900** [langroodi/Adaptive-AUTOSAR#37] [Q&A]
It would be great if you create architectural diagram about various components like diagnostic manager ,
Extended vehicle to understand the project better.

**Comment by langroodi** [comment]
The architecture design is based on the Adaptive AUTOSAR. Please refer to their [official website](https://www.autosar.org/standards/adaptive-platform) to get the related documentation.

The Extended Vehicle is an Adaptive Application based on [Volvo Spec](https://developer.volvocars.com/apis/connected-vehicle/v2/overview/) version 1 (meanwhile Volvo updated the spec to version 2, which can cause some incompatibility while connecting to the API).
